cohortextractor2/concepts/tables.py

```diff
@@ -21,30 +22,40 @@ def __init__(self):
         super().__init__(Table("practice_registrations"))
 
 
-class Patients(TableContract):
+class PatientDemographics(TableContract):
     """
-    The Patients table holds personal details about the patient.
+    Provides demographic information about patients
     """
 
     patient_id = Column(
         type=types.PseudoPatientId(),
-        help=(
+        description=(
             "Patient's pseudonymous identifier, for linkage. You should not normally "
             "output or operate on this column"
         ),
+        help="",
+        constraints=[NotNullConstraint(), UniqueConstraint()],
```

Comment:
@benbc since patient_id isn't necessarily the primary key in a backend table, I think we probably do want to check this?

Reply:
:+1: